cmd/root.go

```diff
@@ -100,6 +102,8 @@ func always(cmd *cobra.Command, args []string) {
 	_ = viper.BindPFlag(config.LoggingLevel, cmd.Flags().Lookup("log"))
 	_ = viper.BindPFlag(config.Server, cmd.Flags().Lookup("server"))
 	_ = viper.BindPFlag(config.Wallet, cmd.Flags().Lookup("wallet"))
+	_ = viper.BindPFlag(config.WalletUser, cmd.Flags().Lookup("walletuser"))
+	_ = viper.BindPFlag(config.WalletPass, cmd.Flags().Lookup("walletpass"))
```

Comment:
Can we make this `walletpassword`? Both the cli and wallet use `-walletpassword=pass`

Reply:
yes